### PR TITLE
Update Grants' form fields

### PR DESCRIPTION
**Pull request by estyxx**
## How to test it

en: https://pycon-git-web-215-update-grants-form-python-italia.vercel.app/en/grants
it: https://pycon-git-web-215-update-grants-form-python-italia.vercel.app/it/grants

### CODE REVIEW

**Review by pauloxnet — APPROVED** [review]

**Review by pauloxnet — APPROVED** [review]

File: frontend/src/locale/index.ts [hunk 1]
```diff
@@ -717,11 +717,11 @@ export const messages = {
     "grants.form.optionalInformation.description":
       "Questo informazioni ci aiutano a rendere la conferenza più inclusiva e diversa.",
 
-    "grants.form.fields.fullName": "Nome completo",
+    "grants.form.fields.fullName": "Nome legale",
     "grants.form.fields.fullName.description":
-      "Il tuo nome completo. Sarà utile nel casi un cui ti daremo un grant.",
+      "Il nome che appare sul passaporto, Carta di Identità o altri documenti legali. Sarà utile nel casi un cui ti daremo un grant.",
```
Comment (marcoacierno) [inline frontend/src/locale/index.ts:722]:
```suggestion
      "Il nome che appare sul passaporto, Carta di Identità o altri documenti legali. Sarà utile nel caso un cui ti daremo un grant.",
```

?

**Review by marcoacierno — COMMENTED** [review]
I think we also need to change this text:

image

the English version says

"Do you need financial aid for travelling to PyCon Italia?"

The Italian version doesn't say it is referring to travel expenses

image

and sounds like it is a request for generic economic support?

**Comment by marcoacierno** [comment]
Can you also run prettier on the files? CI is failing

**Comment by estyxx** [comment]
> I think we also need to change this text:
> 
> image
> 
> the English version says
> 
> "Do you need financial aid for travelling to PyCon Italia?"
> 
> image
> 
> The Italian version is:
> 
> image
> 
> I would make it more clear that it is referring to travel expenses maybe?
> 
> I re-read the message now and I can kind of see it, but I wonder if we should make it more clear

I specified it in the description, I changed in this way: 
`   "Fai click qui se hai bisogno di supporto finanziario per il viaggio",`

**Review by marcoacierno — APPROVED** [review]
🙌  looks good to me

**Pull request merged**